sparks_pif_converters/XRD/xrd_to_pif.py

```diff
@@ -55,21 +56,25 @@ def raw4_txt_to_pif(closed_txt):
 
 
 def raw_to_pif(raw_xrd_file):
+    print("FILE IDENTIFIED AS .raw: {}").format(raw_xrd_file)
 
     # parses .raw file
     try:
-        parsed_raw = BrkRAWParser.parse(raw_xrd_file)
-
         my_pif = ChemicalSystem()
-        my_pif.chemical_formula = raw_xrd_file.split("/")[-1].replace(".raw", "").split("_")[0]
-        my_pif.ids = "".join(raw_xrd_file.split("/")[-1].replace(".raw", "").split("_")[1:])
-
+        my_pif.ids = [os.path.basename(raw_xrd_file).split("_")[0]]
+        my_pif.chemical_formula = os.path.basename(raw_xrd_file).replace(".raw", "").split("_")[1]
```

Comment:
But why replace instead of partition, e.g. `"XXX001_NaCl.raw".rpartition(".")[0]`